src/qtbase-1-fixes.patch

```diff
@@ -78,3 +78,34 @@ index 1111111..2222222 100644
      }
  
      qtConfig(securetransport) {
+diff --git a/mkspecs/common/g++-win32.conf b/mkspecs/common/g++-win32.conf
+index c3a1f3a3..3a2cffaa 100644
+--- a/mkspecs/common/g++-win32.conf
++++ b/mkspecs/common/g++-win32.conf
+@@ -31,7 +31,7 @@ QMAKE_YACCFLAGS         = -d
+ 
+ QMAKE_CFLAGS_SSE2      += -mstackrealign
+ 
+-QMAKE_CXXFLAGS_EXCEPTIONS_ON = -fexceptions -mthreads
```

Comment:
What's mthreads do and what's wrong with it?

Reply:
It's just redundant. I think the assumption is that it pulls either winthreads or pthreads, but our toolchain simply does not recognize it, polluting build logs.

Reply:
Makes sense